api/src/ap/java/com/velocitypowered/api/plugin/ap/PluginAnnotationProcessor.java

```diff
@@ -99,4 +109,8 @@ public synchronized boolean process(Set<? extends TypeElement> annotations,
 
     return false;
   }
+
+  private String getOrElse(Map<String, String> map, String key, String defaultValue) {
+    return map.get(key) == null ? defaultValue : map.get(key);
```

Comment:
Map#getOrDefault ?

Reply:
https://docs.oracle.com/javase/7/docs/api/javax/annotation/processing/ProcessingEnvironment.html

An option with no value returns null which getOrDefault does not cover